src/components/App.js

```diff
@@ -17,22 +17,52 @@ import NotFound from './NotFound/NotFound'
 // router
 import { Route, Switch } from 'react-router'
 import { HashRouter } from 'react-router-dom'
+import { BarLoader } from 'react-spinners'
 
 class App extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      loading: true
+    }
+  }
+  componentDidMount () {
+    // alert('mounted')
+    this.setState({
+      loading: false
```

Comment:
Why we should change this in `componentDidMount`? In my opinion we should set `loading: false` in the constructor